### PR TITLE
Fixes for Github3.py api changes

### PR DESCRIPTION
**Pull request by balloob**
While debugging why issues couldn't be found, I cleaned up the handle_pr method. Found and fixed a bug in github3.py which changed the API, this PR updates for that new api. 

~~Should only merge after sigmavirus24/github3.py#397 has been merged.~~ Fix has been merged into github3.py

### CODE REVIEW

File: farcy/__init__.py [hunk 1]
```diff
@@ -277,34 +271,33 @@ def handle_pr(self, pr):
...
+                file_issues = self.get_issues(pfile)
             except Exception:
                 self.log.exception('Failure with get_issues for {0}'
                                    .format(pfile.filename))
                 exception = True
                 continue
 
-            by_line = defaultdict(lambda: [FARCY_COMMENT_START])
-            process_issues = subtract_issues_by_line(
+            if added is None:
```
Comment (bboe) [inline farcy/__init__.py]: In what cases could `added` be `None`?

Comment (balloob) [inline farcy/__init__.py]: You're right, never.

**Comment by bboe** [comment]
If you want, you can depend on a branch in `setup.py`.

```
dependency_links=[
    'https://github.com/sigmavirus24/betamax/archive/SHASUM.zip#egg=betamax-0.?.?'
]
```

Replace the question marks at the end with the current version (or maybe one version higher?).

**Comment by bboe** [comment]
Great additions! Please merge when tests pass.

**Pull request merged**